diff --git a/config.def.h b/config.def.h
index 1edb647..79adae1 100644
--- a/config.def.h
+++ b/config.def.h
@@ -14,7 +14,7 @@ static const char *colors[SchemeLast][2] = {
 	[SchemeOut] = { "#000000", "#00ffff" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
-static unsigned int lines      = 0;
+static unsigned int lines      = 5;
 
 /*
  * Characters not considered part of a word while deleting words
